clang/test/dpct/cmake_migation/case_015/expected.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.24)
+project(AProject)
+cuda_add_library(target_one MODULE nosrc1.cu)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# No change unless property matches exactly
+set_target_properties(target_one
+                      PROPERTIES
+                      CXX_Standard 14)
+
+# No change unless value is one of 99, 11, 14
+# This negetive test will fail due to limitation in pattern-rewritter
+# set_target_properties(target_one
+#                       PROPERTIES
+#                       CXX_STANDARD 20)
```

Comment:
You can migrate "CXX_STANDARD" to "17" for current migration yaml based rule, and add comments for this rule just for temporary use, and an implicit rule for set_target_properties will be added later on.

Reply:
Done